app/scripts/modules/core/pipeline/config/stages/webhook/modal/addCustomHeader.controller.modal.ts

```diff
@@ -0,0 +1,21 @@
+import {module} from 'angular';
+import {IModalInstanceService} from 'angular-ui-bootstrap';
+
+export class WebhookStageAddCustomHeader {
+  static get $inject() {
+    return ['$scope', '$uibModalInstance'];
+  }
+
+  constructor (private $scope: ng.IScope,
+    private $uibModalInstance: IModalInstanceService) {}
```

Comment:
This can go on the same line as above; otherwise, it should be indented below the other `private` field on the line above.